### PR TITLE
Add suggested namespace for the operators

### PR DESCRIPTION
**Pull request by ShyamsundarR**
Also, correct kustomize to find the right CSV name
post addition of suffix string "operator" to the CSV.

Other additions:
- Add redhat logo
- Update maintainers
- Update description for hub and cluster components
- Update project link to downstream repository

Signed-off-by: Shyamsundar Ranganathan <srangana@redhat.com>

### CODE REVIEW

**Comment by ShyamsundarR** [comment]
@raghavendra-talur and @deepshikhaaa PTAL.

I wonder how the DS builds passed and created the right bundles post the `operator` suffix that was added, any ideas @deepshikhaaa did you override it in your scripts as such?

File: config/hub/manifests/odr/kustomization.yaml [hunk 1]
```diff
@@ -6,6 +6,9 @@ resources:
 
 patches:
 - patch: |-
+    - op: add
+      path: /metadata/annotations/operatorframework.io~1suggested-namespace
+      value: openshift-dr-cluster
```
Comment (raghavendra-talur) [inline config/hub/manifests/odr/kustomization.yaml:11]: Is the value for the namespace correct here?

**Review by raghavendra-talur — APPROVED** [review]

File: config/hub/manifests/odr/kustomization.yaml (hunk 1 above)
Comment (ShyamsundarR) [reply config/hub/manifests/odr/kustomization.yaml:11]: Are you thinking `odr`? We decided that all namespaces start with `openshift` and hence we do not want to shorten it. Is there another problem you see?

File: config/dr-cluster/manifests/odr/kustomization.yaml [hunk 2]
```diff
@@ -6,6 +6,9 @@ resources:
 
 patches:
 - patch: |-
+    - op: add
+      path: /metadata/annotations/operatorframework.io~1suggested-namespace
+      value: openshift-dr-cluster
     - op: replace
       path: /metadata/name
       value: odr-cluster.v0.0.0
```
Comment (ShyamsundarR) [inline config/dr-cluster/manifests/odr/kustomization.yaml:14]: I think this needs to change as well to `odr-cluster-operator` @raghavendra-talur

Comment (raghavendra-talur) [reply config/dr-cluster/manifests/odr/kustomization.yaml:14]: I agree, this needs to change too.

File: config/hub/manifests/odr/kustomization.yaml [hunk 3]
```diff
@@ -6,6 +6,9 @@ resources:
...
     - op: replace
       path: /metadata/name
       value: odr-hub.v0.0.0
```
Comment (raghavendra-talur) [inline config/hub/manifests/odr/kustomization.yaml:14]: We should add a operator suffix here too.

File: config/hub/manifests/odr/kustomization.yaml (hunk 1 above)
Comment (raghavendra-talur) [reply config/hub/manifests/odr/kustomization.yaml:11]: If the namespace is "openshift-dr-cluster" for both the hub and cluster operators then this is correct. No other issue.

Comment (ShyamsundarR) [reply config/hub/manifests/odr/kustomization.yaml:11]: Ack! Changing this to `openshift-dr-system` namespace across both hub and cluster bundles.

**Comment by ShyamsundarR** [comment]
@raghavendra-talur All updates are complete. PTAL and based on the review outcome we can merge (or amend). Thanks!

**Review by raghavendra-talur — APPROVED** [review]

**Pull request merged**